cli_tools_e2e_test/gce_image_import_export/test_suites/import/cli_tests.go

```diff
@@ -132,6 +137,39 @@ func runImageImportOSTest(ctx context.Context, testCase *junitxml.TestCase, logg
 	runImportTest(ctx, argsMap[testType], testType, testProjectConfig, imageName, logger, testCase)
 }
 
+func runImportForImage(sourceImageURI, osID string) func(
+	context.Context, *junitxml.TestCase, *log.Logger, *testconfig.Project, utils.CLITestType) {
+
+	return func(ctx context.Context, testCase *junitxml.TestCase, logger *log.Logger,
+		testProjectConfig *testconfig.Project, testType utils.CLITestType) {
+		suffix := path.RandString(5)
+		imageName := "e2e-test-image-import-os-" + suffix
+
+		argsMap := map[utils.CLITestType][]string{
+			utils.Wrapper: {"-client_id=e2e",
+				"-project", testProjectConfig.TestProjectID,
+				"-image_name", imageName,
+				"-os", osID,
+				"-source_image", sourceImageURI,
+				"-zone", testProjectConfig.TestZone,
+			},
+			utils.GcloudProdWrapperLatest: {"beta", "compute", "images", "import", imageName, "--quiet",
+				"--docker-image-tag=latest", "--os", osID, "--project", testProjectConfig.TestProjectID,
+				"--source-image", sourceImageURI,
+				"--zone", testProjectConfig.TestZone,
+			},
+			utils.GcloudLatestWrapperLatest: {"beta", "compute", "images", "import", imageName, "--quiet",
+				"--docker-image-tag=latest", "--os", osID, "--project", testProjectConfig.TestProjectID,
```

Comment:
Just noticed all tests here are "beta". Is that suppose to be? If not we may need to make it an arg just as Zoran did in another PR today

Reply:
--docker-image-tag is in beta only. If a test needs to use the latest wrapper, we have to use beta gcloud. 

Can we have --docker-image-tag in GA, still hidden?

Reply:
Good discussion -- I realized I actually _don't_ want to use the latest wrapper, since this is intended to verify that gcloud will work once pushed.

Updated to remove `--docker-image-tag=latest`